app/models/cloud_volume_snapshot.rb

```diff
@@ -26,6 +26,61 @@ def my_zone
     self.class.my_zone(ext_management_system)
   end
 
+  def self.create_snapshot_queue(userid, cloud_volume, options = {})
+    ext_management_system = cloud_volume.&ext_management_system
+    task_opts = {
+      :action => "creating volume snapshot in #{ext_management_system.inspect} for #{cloud_volume.inspect} with #{options.inspect}",
+      :userid => userid
+    }
+
+    queue_opts = {
+      :class_name  => cloud_volume.class.name,
+      :instance_id => cloud_volume.id,
+      :method_name => 'create_volume_snapshot',
+      :priority    => MiqQueue::HIGH_PRIORITY,
```

Comment:
Probably don't need this to be HIGH_PRIORITY, realize this was just a copy but openstack had a habit of making everything high-priority when it wasn't really necessary

Reply:
assuming across the board